modules/shortcodes/js/blocks/vr-block.jsx

```diff
@@ -53,35 +54,36 @@
 					<div>
 						<Placeholder
 							key="placeholder"
-							instructions={ __( 'Enter URL to VR image', 'jetpack' ) }
 							icon="format-image"
 							label={ __( 'VR Image', 'jetpack' ) }
 							className={ props.className }
 						>
-							<UrlInput
+							<TextControl
```

Comment:
Any good reason to use `TextControl` here? Is it because of the `style` prop?

Reply:
What would be the alternative? As @simison noted in the PR desc, `UrlInput` is deprecated...

Reply:
We could use `<input>` like core-embed block does:

https://github.com/WordPress/gutenberg/blob/0063c37c5a470ec246704608623b19c10f3301fc/packages/block-library/src/embed/index.js#L304-L310

Reply:
Yeah, that might make sense.